src/compressions/ccitt.zig

```diff
@@ -0,0 +1,326 @@
+const std = @import("std");
+const Image = @import("../Image.zig");
+const ImageUnmanaged = @import("../ImageUnmanaged.zig");
+
+const Code = struct {
+    run_length: u16,
+    code: u16,
+    code_length: u8,
+};
+
+pub const Color = enum(u1) {
+    black = 0,
+    white = 1,
+};
+
+pub const white_terminating_codes = [_]Code{
+    // zig fmt: off
```

Comment:
Same here, if you put a comma at the end of the last entry, you won't need zig fmt off